### ISSUE TITLE
Tank got a Speed buff

### ISSUE THREAD

**Issue by C0REX** [SirPlease/L4D2-Competitive-Rework#92]
sometimes the tank gets a speed buff in versus mode. I added this in the cfg.
confogl_addcvar z_tank_speed_vs 210 // default 210
confogl_addcvar z_tank_walk_speed 100 // default 100
But no difference. It comes as surprising and unpredictable as the bug with the jockey in my post before. The only difference is that it only happens on some maps like Deathtoll or Center. Why do I think it's not a hack from other players? Because I suddenly had the speed buff even as a tank and I didn't use hacks or scripts like this. Sad that a lot of things don't work. :(

**Comment by C0REX** [comment]
bug is gone after delete l4d2_slowdown_control_eq.smx

**Comment by SirPlease** [comment]
After reading your last post, it seems as if you're using a few of these plugins or you're adding plugins that interfere with what the plugins in this repository are trying to do.
Issues like the ones you've reported so far are never seen on the competitive servers that all run their installs (including my own) based on this github.

I can't provide support if you're working outside of the scope of this repository and don't give me any additional details on your setup.

**Comment by C0REX** [comment]
Hey, I installed the repo on a fresh server. A fixed config is loaded via sm_forcematch. So I added the following to the server.cfg from the repo:

confogl_addcvar stripper_cfg_path cfg / stripper / pmelite
sm_forcematch pmelite

You are right I have 2 plugins outside of your repo. These 2 plugins were placed in the optional folder and loaded via confogl_plugins.cfg. "tank_swap.smx" and "votemanager2.smx". Deactivating the two plugins unfortunately made no difference.

I deactivated some plugins from the repo and put them in the disabled folder. In addition I have excluded them from confogl_plugins.cfg and should not be loaded. [disabled folder](http://tensaiberlin.bplaced.net/disable_folder.jpg)

Cvars for the deactivated plugins are excluded from the respective cfg's. Some Stipper files adapted, partly excluded.

I can only imagine that it is because some plugins have been deactivated. I know the repo is meant to run as a whole and many plugins are networked with each other. But since everything works really well except the bug, I thought it was probably something else. because the bug with the jockey does not occur permanently. Which makes the search for the bug very difficult. We also observed that the jockey does not always port his victim into the skybox, Sometimes both remain on the map. It is also only ported in x or y .. difficult to say. Z remains the same.

If the error only occurs to me, it is probably due to the handling of your repo and not the repo itself. In this case, sorry for the excitement :)

But maybe you have an idea which of the deactivated plugins can be responsible, or a clue where we can search?
